backend/app/controllers/application_controller.rb

```diff
@@ -2,4 +2,5 @@
 
 class ApplicationController < ActionController::Base
   # before_action :authenticate_user!
+  protect_from_forgery with: :null_session
```

Comment:
We need to discuss whether we actually need CSRF protection, I don't think so since we're using JWTs but I'm no expert, and there are other security considerations there